### DISCUSSION TITLE
/contents/ API 404 when file path has a space

### DISCUSSION THREAD

**Discussion by eberg23** [community/community#14278] [Other Feature Feedback, Questions, & Ideas] [answered]
Initially, I'm making this call to the pulls endpoint.
```GET https://api.github.com/repos/AcmeIndustries/anvil/pulls/2/files?per_page=3000```

In the response of that, I get info about each file in the pull, which looks like this
```
{
  'sha': 'a575c41e5725d883f027669da0bfc68eda21d3cd', 
  'filename': 'bugs bunny.txt', 
  'status': 'modified', 
  'additions': 1, 
  'deletions': 0, 
  'changes': 1, 
  'blob_url': 'https://github.com/AcmeIndustries/anvil/blob/93db020906653e3b10458a2c6fc2442623c7b088/bugs+bunny.txt',
  'raw_url': 'https://github.com/AcmeIndustries/anvil/raw/93db020906653e3b10458a2c6fc2442623c7b088/bugs+bunny.txt',
  'contents_url': 'https://api.github.com/repos/AcmeIndustries/anvil/contents/bugs+bunny.txt?ref=93db020906653e3b10458a2c6fc2442623c7b088', 
  'patch': '@@ -1,2 +1,3 @@\n hello world\n+hello yosemite sam\n '
}
```

So I grab the `contents_url` from there and call it:

```
GET https://api.github.com/repos/AcmeIndustries/anvil/contents/bugs+bunny.txt?ref=93db020906653e3b10458a2c6fc2442623c7b088
```

=> error 404: `b'{"message":"Not Found","documentation_url":"https://docs.github.com/rest/reference/repos#get-repository-content"}'`

Expected: 200 + file contents in response, per the API specifications.

This appears to be a recent regression. Saw this for the first time on April 1.

**Comment by alhazen11** [comment]
I have some issues here.

**Reply by eberg23** [reply]
@alhazen11 Issues with my write-up, or with the issue it describes?

**Reply by alhazen11** [reply]
I have a problem same with the your issue describes

**Comment by eberg23** [comment]
Do these get looked at and turned into issues that can be tracked?

**Reply by queenofcorgis** [reply]
It varies post to post, but if there is an Issue that users can follow along, usually the staff member who is working on it will post a link to it and any relevant updates. However, not all posts with have an associated Issue, hope that helps answer your question!

**Reply by eberg23** [reply]
It answers my question. But the bigger question is, when will this be fixed? It's an easily reproducible bug affecting multiple customers.

**Comment by eberg23** [comment, marked as answer]
Thanks to whoever fixed this!

**Reply by terwer** [reply]
Still not fixed now
